src/Analyzer.PowerShellRuleEngine/PowerShellRuleEngine.cs

```diff
@@ -85,8 +85,10 @@ public IEnumerable<IEvaluation> AnalyzeTemplate(TemplateContext templateContext)
                     {
                         evaluationResults.Add(new PowerShellRuleResult(false, lineNumber));
                     }
+
+                    var ruleId = (executionResult.Name as string)?.Replace(" ", "") ?? string.Empty;
                     var ruleDescription = executionResult.Name + ". " + uniqueError.Key;
-                    evaluations.Add(new PowerShellRuleEvaluation("", ruleDescription, false, evaluationResults));
+                    evaluations.Add(new PowerShellRuleEvaluation(ruleId, ruleDescription, false, evaluationResults));
```

Comment:
Should we add an issue for the "longer-term" solution? To cover fixing the rule ids, rule description, and handling instance-specific information for both engines? I can add it

Reply:
It's covered in #199.